### ISSUE TITLE
[Bug]: LIQA currently crashes with most recent version of lifelines package

### ISSUE THREAD

**Issue by jpfry327** [ylab-hi/ScanExitronLR#2]
### Contact Details

joshua.fry@northwestern.edu

### What happened?

We use LIQA for estimating affected transcript abundances. Currently, LIQA will crash with the python lifelines package > v0.27.0.

If you are experiencing this, please downgrade your lifelines package to 0.26.4

### Version

1.0.0 (Default)

### What platform are you working on?

Mac Intel

### Relevant log output

```shell
Traceback (most recent call last):
  File "/panfs/home/yang4414/fryxx094/miniconda3/envs/selr2_test/lib/python3.9/site-packages/lifelines/utils/__init__.py", line 966, in pass_for_numeric_dtypes_or_raise_array
    raise ValueError("Empty array/Series passed in.")
ValueError: Empty array/Series passed in.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/panfs/home/yang4414/fryxx094/miniconda3/envs/selr2_test/lib/python3.9/site-packages/liqa_src/quantify.py", line 419, in <module>
    kmf.fit(Tforkmf, event_observed=Eforkmf)
  File "/panfs/home/yang4414/fryxx094/miniconda3/envs/selr2_test/lib/python3.9/site-packages/lifelines/utils/__init__.py", line 56, in f
    return function(model, *args, **kwargs)
  File "/panfs/home/yang4414/fryxx094/miniconda3/envs/selr2_test/lib/python3.9/site-packages/lifelines/fitters/kaplan_meier_fitter.py", line 127, in fit
    return self._fit(durations, event_observed, timeline, entry, label, alpha, ci_labels, weights)
  File "/panfs/home/yang4414/fryxx094/miniconda3/envs/selr2_test/lib/python3.9/site-packages/lifelines/fitters/kaplan_meier_fitter.py", line 354, in _fit
    (self.durations, self.event_observed, self.timeline, self.entry, self.event_table, self.weights) = _preprocess_inputs(
  File "/panfs/home/yang4414/fryxx094/miniconda3/envs/selr2_test/lib/python3.9/site-packages/lifelines/utils/__init__.py", line 906, in _preprocess_inputs
    durations = np.asarray(pass_for_numeric_dtypes_or_raise_array(durations)).reshape((n,))
  File "/panfs/home/yang4414/fryxx094/miniconda3/envs/selr2_test/lib/python3.9/site-packages/lifelines/utils/__init__.py", line 970, in pass_for_numeric_dtypes_or_raise_array
    raise ValueError("Values must be numeric: no strings, datetimes, objects, etc.")
ValueError: Values must be numeric: no strings, datetimes, objects, etc.
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by cauliyang** [comment]
I have fixed the version of lifelines to 0.26.4 so that users do not need to worry about that.

Please sync current changes to pypi. Later, I will create github actions to upload PyPI automatically after we change something.

**Comment by jpfry327** [comment]
Changes synced to PyPi
